testing/features/oisc_warnings.feature

```diff
@@ -7,23 +7,38 @@ Feature: OISC Warning components
     Background:
       Given a Title and Description OISC component is on the page
 
-    Scenario: OISC component has a title and message
+    Scenario: English OISC component has a title and message
```

Comment:
The part of OISC that's translated is the "Close this warning" label, the rest of the content is user supplied, so I don't believe this is the right set of tests for OISC.

Reply:
I couldn't find that label anywhere on the current version. So I'm guessing that's a new piece of functionality?